cli/src/pcluster/schemas/imagebuilder_schema.py

```diff
@@ -0,0 +1,128 @@
+from marshmallow import fields, post_load
+
+from pcluster.config.imagebuilder_config import (
+    ImageBuilderBucketLogConfig,
+    ImageBuilderBuildConfig,
+    ImageBuilderChefCookbookConfig,
+    ImageBuilderComponentsConfig,
+    ImageBuilderConfig,
+    ImageBuilderDevSettingsConfig,
+    ImageBuilderImageConfig,
+    ImageBuilderVolumeConfig,
+)
+from pcluster.schemas.cluster_schema import BaseSchema
+
+
+class ImageBuilderVolumeSchema(BaseSchema):
```

Comment:
You can avoid the ImageBuilder prefix in the classes too. It's clear that we're talking about image builder components because we're in the imagebuilder_schema.py module.

Reply:
Done